From 1e416d74a87292a31fcdded106b6bb7c5f8bfdbb Mon Sep 17 00:00:00 2001
From: Fabio Estevam <fabio.estevam@nxp.com>
Date: Thu, 3 May 2018 13:21:41 -0300
Subject: [PATCH 3825/5242] MLK-18339: spi: imx: Allow driver to be built on
 ARM64 i.MX

commit  58475b8e0f798054d4b72ecf30cfedebe9986b35 from
https://source.codeaurora.org/external/imx/linux-imx.git

The spi_imx IP block is also present on i.MX8M, so allow it
to be built for this platform.

Signed-off-by: Fabio Estevam <fabio.estevam@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/spi/Kconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/spi/Kconfig b/drivers/spi/Kconfig
index 2f1315e..fa04c02 100644
--- a/drivers/spi/Kconfig
+++ b/drivers/spi/Kconfig
@@ -276,7 +276,7 @@ config SPI_IMG_SPFI
 
 config SPI_IMX
 	tristate "Freescale i.MX SPI controllers"
-	depends on ARCH_MXC || COMPILE_TEST
+	depends on ARCH_MXC || ARCH_MXC_ARM64 || COMPILE_TEST
 	select SPI_BITBANG
 	help
 	  This enables using the Freescale i.MX SPI controllers in master
-- 
1.7.9.5

